ourrevolution/settings.py

```diff
@@ -86,6 +86,10 @@
     'EVENTS_PROMOTE_RECENT_CUTOFF_DAYS',
     14
 ))
+EVENTS_PROMOTE_SENDABLE_CONS_GROUP_ID = str(os.environ.get(
```

Comment:
this var is named after events app but only used in bsd app. maybe update naming convention to be consistent? `BSD_`?

Reply:
actually it makes sense for this to be a event promo settings, but it should be invoked in events app or organizing hub, and then passed in bsd app as a param to the find method, right?

Reply:
Agreed.